### ISSUE TITLE
Update completions after manually triggering them

### ISSUE THREAD

**Issue by LukasPietzschmann** [ms-jpq/coq_nvim#616]
So I've set `coq_settings.completion.always` to `false`. But when I invoke completions and type more letters, the completions do not update automatically.

I've looked through issues and the documentation, but I could not find a way to make the completions update automatically. Is this not possible, or am I missing something?

Cheers! Lukas :)

**Comment by Watynecc** [comment]
I think we're in the same track
I found out about https://github.com/ms-jpq/coq_nvim/blob/coq/docs/COMPLETION.md but it doesn't help
Here's my coq.settings
```
vim.g.coq_settings = {
    auto_start = 'shut-up',
    completion = {
        always = false
    },
    clients = {
        tmux = { enabled = false },
        buffers = {
            enabled = false,
        }
    },
}
```

**Comment by abc123098123** [comment]
I agree. I am using `always = false` and I want the results to update automatically after typing more characters. I find this useful because I am having an issue where, even after completing, the same string still appears as a result.

**Comment by Watynecc** [comment]
Man I don't know how to contribute yet but I will find a way it mustn't be
that hard

Le sam. 6 juil. 2024, 03:44, Romeu Vieira ***@***.***> a
écrit :

**Comment by Watynecc** [comment]
@RomeuG Any ideas where's the section of the code that imply using ctrl space/Manual completion ?

**Comment by abc123098123** [comment]
> @RomeuG Any ideas where's the section of the code that imply using ctrl space/Manual completion ?

No clue, really. I would try to understand the keybind, how it is set, and how it is triggered, I guess.

**Comment by Watynecc** [comment]
Crap... I don't have any plan about how to fix it...

Le 11 juillet 2024 15:45:37 GMT+02:00, Romeu Vieira ***@***.***> a écrit :

**Comment by Watynecc** [comment]
Thanks you #687 !!! Now it works !
